src/acp/rule.ts

```diff
@@ -73,6 +87,7 @@ export function removeRequiredRuleFromPolicy(
  * ```
  *
  * Overwrites the rule refining the scope of a given the [[Policy]]. If an agent
```

Comment:
Forgot to remove the old version :) (Also on line 159.)

```suggestion
```

Reply:
Good catch, fixed in 62a05e84592e8cf4e65882f578ef1282565c1894